samples/openthread/coap_client/README.rst

```diff
@@ -318,7 +318,7 @@ Device Firmware Upgrade using mcumgr
 Sample output
 =============
 
-You can observe the sample logging output through a serial port.
+You can observe the sample logging output through a serial port, |ANSI|.
```

Comment:
```suggestion
You can observe the sample logging output through a serial port |ANSI|.
```
"a serial port that supports VT100/ANSI escape characters" sounds odd. Should this part and CoAP server also mention terminal emulator?